examples/aws/resourcerecordset/v1alpha1/claim-using-zone-with-labels.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: route53.aws.crossplane.io/v1alpha1
+kind: ResourceRecordSet
```

Comment:
Shouldn't we be using AWSResourceRecordSetV1alpha1 rather than the direct AWS resource here?

Reply:
you're right!